src/components/badge/beta_badge/_beta_badge.scss

```diff
@@ -60,16 +60,6 @@
   }
 }
 
-.ouiBetaBadge--singleLetter {
-  padding: 0 0 0 1px;
-  width: $ouiSizeL;
-
-  &.ouiBetaBadge--small {
-    width: $ouiSize + $ouiSizeXS;
-    padding: 0 0 0 1px;
-  }
-}
-
```

Comment:
Should we note somewhere that we're not actually preventing a user from providing a single character label - we've just removed custom styling for that scenario and encouragement of that usage pattern from the doc. But have we validated the look/behavior of a single letter label without these styles? Or do we actually want to add some minimum length enforcement within the component (that prevents rendering?)